packages/salesforcedx-vscode/CHANGELOG.md

```diff
@@ -1,3 +1,21 @@
+# 48.24.0 - July 8, 2020
+
+## Fixed
+
+#### salesforcedx-vscode-core
+
+- Cancelling a command execution should not throw errors ([PR #2316](https://github.com/forcedotcom/salesforcedx-vscode/pull/2316))
```

Comment:
@rcoringrato-sfdc , we had a similar entry in last week's log. Specify the error type to make it clearer?
Fix SFDX commands causing _toString_ errors when cancelled

Reply:
Yes, this is a follow up fix to last week's one.